query_server/sqllogicaltests/src/instance.rs

```diff
@@ -101,7 +101,12 @@ pub fn construct_write_url(options: &SqlClientOptions) -> Result<Url> {
     http_query.push_str(tenant.as_str());
 
     http_query.push_str("&precision=");
-    http_query.push_str(precision.as_ref().map(|s| s.as_str()).unwrap_or("NS"));
+    http_query.push_str(
+        precision
+            .as_ref()
+            .map(|s| s.as_str())
+            .unwrap_or_else(|| "NS"),
```

Comment:
The static type of string( &'static str ) "NS" doesn't need to be lazily evaluated.